extensions/macrumors/package.json

```
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "macrumors",
+  "title": "MacRumors",
+  "description": "Browse MacRumors headlines from the comfort of Raycast.",
+  "icon": "macrumors-extension-icon.png",
+  "author": "hughmcmillanv",
+  "categories": [
+    "News"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "macrumors",
+      "title": "MacRumors",
```

```suggestion
      "title": "Show Latest MacRumors",
```
Or something like that